internal/provider/providers/route53/provider.go

```diff
@@ -1,20 +1,19 @@
 package route53
 
 import (
-	"bytes"
 	"context"
 	"encoding/json"
-	"encoding/xml"
 	"fmt"
 	"net/http"
 	"net/netip"
-	"net/url"
-	"time"
+
+	"github.com/aws/aws-sdk-go/aws/credentials"
```

Comment:
Go SDK v1 is EOL https://aws.amazon.com/blogs/developer/announcing-end-of-support-for-aws-sdk-for-go-v1-on-july-31-2025/

You're gonna want to use `github.com/aws/aws-sdk-go-v2/`